### PR TITLE
Add docs and CI

### PR DESCRIPTION
**Pull request by lightclient**
Adds a small amount of documentation and CI! >:-)

### CODE REVIEW

File: etk-analyze/Cargo.toml [hunk 1]
```diff
@@ -1,17 +1,18 @@
 [package]
-name = "gencfg-analyze"
+name = "etk-analyze"
 version = "0.1.0"
 authors = ["Sam Wilson <sam.wilson@mesh.xyz>"]
+license = "MIT OR Apache-2.0"
```
Comment (SamWilsn) [inline etk-analyze/Cargo.toml:5]: I'm an MPL or AGPL fan at heart...

File: .github/workflows/test.yaml [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+jobs:
+  check:
+    name: Check
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions-rs/toolchain@v1
+        with:
+          profile: minimal
+          toolchain: nightly
```
Comment (SamWilsn) [inline .github/workflows/test.yaml:14]: Should we pick a particular minimum rust version to support?

Comment (lightclient) [reply .github/workflows/test.yaml:14]: I don't have a preference! Let me know what you'd like to do.

Comment (SamWilsn) [reply .github/workflows/test.yaml:14]: Nightly is fine for now.

File: README.md [hunk 3]
```diff
@@ -0,0 +1,122 @@
...
+; Read the calldata into memory.
+calldatasize                    ; [calldatasize]
+push1 0                         ; [calldatasize, calldata_ptr]
+dup1                            ; [calldatasize, calldata_ptr, mem_ptr]
+calldatacopy                    ; []
+
+; Extract only the function selector
+push1 0                         ; [selector_ptr]
+mload                           ; [dirty_selector]
+push1 e0                        ; [dirty_selector, 28*8]
```
Comment (SamWilsn) [inline README.md:85]: Would you mind updating the docs with the `0x` prefix?

**Review by SamWilsn — CHANGES_REQUESTED** [review]

File: README.md (hunk 3 above)
Comment (lightclient) [reply README.md]: Accidentally pushed that commit to this PR :) I updated the doc anyways.

**Pull request merged**